SuperMario-Tweaker.sh

```
@@ -1,99 +1,250 @@
-ro.hwui.layer_cache_size=48
-ro.hwui.path_cache_size=32
-ro.hwui.r_buffer_cache_size=8
-ro.hwui.drop_shadow_cache_size=6
-ro.hwui.gradient_cache_size=1
-ro.hwui.texture_cache_flushrate=0.4
-ro.hwui.text_small_cache_width=1024
-ro.hwui.text_small_cache_height=512
-ro.hwui.text_large_cache_width=2048
-ro.hwui.text_large_cache_height=1024
-EOF
+        if cat <<EOF >> "$MODPATH"/system.prop; then
```

Shellcheck (0.10.0)

[error] Couldn't parse this here document. Fix to allow more checks.

(SC1073)